.github/ISSUE_TEMPLATE/other.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Other
+about: Questions and anything that isn't a feature or a bug
```

Comment:
For questions, shouldn't we propose in the request body for people to visit Discord?